src/Utility/Configuration.php

```diff
@@ -67,6 +68,16 @@ public static function fromRemote(): Configuration
 
     public static function fromLocal(): Configuration
     {
+        if (file_exists(getcwd() . '/app/config/parameters.yml')) {
+            $path = getcwd() . '/app/config/parameters.yml';
```

Comment:
Maybe beter to create the variable above the if-statement and use the variable in the file_exists.